index.d.ts

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
nit: can you remove this newline? happy to merge & publish after that 👍

Reply:
Removed. Thanks.